projects/Mallard/src/components/layout/ui/errors/error-boundary.tsx

```diff
@@ -22,6 +29,9 @@ class ErrorBoundary extends Component<
 
     render() {
         if (this.state.hasError) {
+            if (this.props.error) {
+                return this.props.error
+            }
```

Comment:
not of fan of duplicating `message` status versus `error` prop, instead I'd say let's design the code so that we throw whichever error needs to be shown from within the error boundary and use the normal logic of this component

Reply:
agreed! I'm thinking for the future, when using suspense, I wanna err on the side of extensibility for now and then when we have a better idea of how it's consumed (have a look at our cachedOrPromise rejections now, they are all a bit different) we can unify them as props